test/OpenTelemetry.Extensions.Tests/OpenTelemetry.Extensions.Tests.csproj

```diff
@@ -18,6 +18,7 @@
   <ItemGroup>
     <PackageReference Include="OpenTelemetry.Exporter.InMemory" Version="$(OpenTelemetryCoreLatestVersion)" />
     <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.10.0" />
+    <PackageReference Include="Moq" Version="4.20.72" />
```

Comment:
Please remove usage of Moq. See https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1467.